docs/docs/admin/monitoring/logging/explorer.md

```diff
@@ -0,0 +1,119 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Grafana Explorer
+
+The Grafana Explorer can be used to view the logs of different components, such
+as the CCM backend or user sessions. The logs are stored persistently, so the
+administrator can also view the logs of already terminated sessions, for
+example to investigate an unexpected session termination.
+
+The Grafana Explorer is available to administrators and can be accessed from
+the Grafana link in the main menu. Select "Explore" to access the log explorer:
+
+![Explore in the main Grafana menu](./explore.png){:style="width:300px"}
+
+## Select the Correct Source
+
+Before you can view logs, you must make sure that the correct source is
+selected for the CCM. For this please select **Loki (Capella Collaboration
+Manager)** in the data source picker.
+
+![Select CCM in Data Source Picker](./data_source_picker.png){:style="width:500px"}
+
+## Labels
+
+The application logs are stored in loki, which uses labels to store and
+retrieve logs efficiently. This means that each log in loki has a set of labels
+assigned to it, where a label is essentially a key-value pair.
+
+The Explorer now consists of two main components, which are described below.
```

Comment:
I think you're referring to Label Filters and Operations here but it's not very clear. Consider making the levels of both Component's headings the same, them currently being different is confusing. You might also want to just name the components here.

Reply:
I removed the sentence since I'd neither call "Label filters" nor "Operations" a component.